config/locales/en.yml

```diff
@@ -361,8 +361,8 @@ en:
       benefits:
         part_or_full_time: "Answer cannot be ‘all’ for income from Universal Credit, state pensions or benefits if the tenant or their partner works part-time or full-time"
       earnings:
-        over_hard_max: "Net income cannot be greater than %{hard_max} per week given the household’s working situation"
-        under_hard_min: "Net income cannot be less than %{hard_min} per week given the household’s working situation"
+        over_hard_max: "The household's income cannot be greater than %{hard_max} per week given the household’s working situation"
+        under_hard_min: "The household's income cannot be less than %{hard_min} per week given the household’s working situation"
```

Comment:
Would this need to be household’s instead of household's ?

Reply:
Drat, yes